testdata/spinnaker-monitoring.yml

```diff
@@ -0,0 +1,27 @@
+datadog:
+  enabled: true
```

Comment:
Each individual metric store's `enabled` field is technically unnecessary to pass to spinnaker-monitoring since it is only using the `monitor.metric_store` list to determine which stores are enabled. That said, it seemed unnecessarily complex to explicitly remove `enabled` from each store in the output, either by manually nulling it out or by creating separate output protos for each store without that field.

Reply:
Is one option here to just remove it from the proto entirely so that it gets ignored on input and output?

Reply:
Oh well we do need it in the input, then the translation layer writes each enabled type to `monitor.metric_store`.

Reply:
Oh, right of course, I just looked at that code.  I think we should leave this how you have it now!